python/ray/train/huggingface/accelerate/accelerate_trainer.py

```diff
@@ -34,6 +35,15 @@
     from ray.tune.trainable import Trainable
 
 
+ACCELERATE_TRAINER_DEPRECATION_MESSAGE = (
+    "The AccelerateTrainer is deprecated. "
+    "Use TorchTrainer instead. "
```

Comment:
```suggestion
    "The AccelerateTrainer will be hard deprecated in Ray 2.8. "
    "From Ray 2.7, please use TorchTrainer instead. "
```

Does that make sense?